packages/core/test/todo.md

```diff
@@ -0,0 +1,2022 @@
+//     content: "<div class=\"unbreakable\"><a class=\"btn\" href=\"#\">&#65279;dom to edit&#65279;</a></div>",
+//     steps: [{
+//         start: "a:contents()[0]->13",
+//         key: 'ENTER',
+//     }, {
+//         key: 'a',
+//     }],
+//     test: {
+//         content: "<div class=\"unbreakable\"><a class=\"btn\" href=\"#\">dom to edit</a><a class=\"btn\" href=\"#\">&#65279;a&#65279;</a></div>",
+//         start: "a:eq(1):contents()[0]->2",
+//     },
+// },
+// {
+//     name: "in div > button.btn: ENTER -> 'a' at end (after invisible space)",
+//     content: "<div class=\"unbreakable\"><button class=\"btn\" href=\"#\">&#65279;dom to edit&#65279;</button></div>",
+//     steps: [{
+//         start: "button:contents()[0]->13",
+//         key: 'ENTER',
+//     }, {
+//         key: 'a',
+//     }],
+//     test: {
+//         content: "<div class=\"unbreakable\"><button class=\"btn\" href=\"#\">dom to edit</button><button class=\"btn\" href=\"#\">&#65279;a&#65279;</button></div>",
+//         start: "button:eq(1):contents()[0]->2",
+//     },
+// },
+// {
+//     name: "in div > a.btn: ENTER -> 'a' on partial selection",
+//     content: "<div class=\"unbreakable\"><a class=\"btn\" href=\"#\">&#65279;dom to edit&#65279;</a></div>",
+//     steps: [{
+//         start: "a:contents()[0]->4",
+//         end: "a:contents()[0]->8",
+//         key: 'ENTER',
+//     }, {
+//         key: 'a',
+//     }],
+//     test: {
+//         content: "<div class=\"unbreakable\"><a class=\"btn\" href=\"#\">dom</a><a class=\"btn\" href=\"#\">&#65279;aedit&#65279;</a></div>",
+//         start: "a:eq(1):contents()[0]->2",
+//     },
+// },
+// {
+//     name: "in div > a.btn: ENTER -> 'a' on selection of all visible text",
+//     content: "<div class=\"unbreakable\"><a class=\"btn\" href=\"#\">dom to edit</a></div>",
+//     steps: [{
+//         start: "a:contents()[0]->0",
+//         end: "a:contents()[0]->11",
+//         key: 'ENTER',
+//     }, {
+//         key: 'a',
+//     }],
+//     test: {
+//         content: "<div class=\"unbreakable\"><a class=\"btn\" href=\"#\">Label</a><a class=\"btn\" href=\"#\">&#65279;a&#65279;</a></div>",
+//         // Removing all text in a link replaces that text with "Label"
+//         start: "a:eq(1):contents()[0]->2",
+//     },
+// },
+// {
+//     name: "across 2 a.btn: ENTER on selection across two a.btn",
+//     content: "<div class=\"unbreakable\"><a class=\"btn\" href=\"#\">dom not to edit</a><a class=\"btn\" href=\"#\">other dom not to edit</a></div>",
+//     steps: [{
+//         start: "a:contents()[0]->0",
+//         end: "a:eq(1):contents()[0]->11",
+//         key: 'ENTER',
+//     }],
+//     test: {
+//         content: "<div class=\"unbreakable\"><a class=\"btn\" href=\"#\">Label</a><a class=\"btn\" href=\"#\">&#65279;ot to edit&#65279;</a></div>",
+//         start: "a:eq(1):contents()[0]->1",
+//     },
+// },
```

Comment:
Why are these tests commented out, while the others are just "there" ?

Reply:
https://github.com/odoo-dev/jabberwock/pull/46/files#diff-0875bb23b2e3171746454f3566139fbaR1072